docs/projects/fibonacci/_posts/_posts/2019-10-04-javascript.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Fibonacci sequence in JavaScript  
+layout: default  
+last-modified: 2019-10-01 4
+featured-image:  
+tags: [javascript, fibonacci]  
+authors:[ Alina1Black ]
+
+---
+
+In this article, we will write an algorithm of the Fibonacci sequence in JavaScript.
+
+## How to Implement the Solution
+
+Let's look at the code in detail:  
+
+code for fibonacci.js:-  
+
+```javascript
+function fibonacci(num) {
+    let n = Number(num);
+    let first = 0;
+    let second = 1;
+    let result = 0;
+    for (let i = 1; i <= n; i++) {
+    	result = first + second;
+    	first = second;
+    	second = result;
+        console.log(i + ": " + first);
+    }
+}
+
+num = 10; //we want to print the first 10 Fibonacci numbers
+
+if (num && !isNaN(num)) {
+    fibonacci(num);
+} else {
+    console.log("Usage: please input the count of fibonacci numbers to output")
+}
+
+```
```

Comment:
This doesn't look like the same solution we have in the repo: https://github.com/TheRenegadeCoder/sample-programs/blob/master/archive/j/javascript/fibonacci.js

Reply:
This is a simpler version that runs on the browser instead of requiring NodeJS. Should I re-edit the example? or should I revise the Article I just wrote? Either is fine!

Reply:
Fixed it! Added both examples in the article.